doc/pictures/testclient_basic.cast

```diff
@@ -1,22 +1,22 @@
 {"version": 2, "width": 80, "height": 33, "ts": 1599119755, "env": {"SHELL": "/usr/bin/fish", "TERM": "rxvt-unicode-256color"}}
 [0.028694, "o", "\u001b]0;fish /home/wenwen/tools/kuksa.val"]
-[0.029339, "o", "/kuksa_viss_client\u0007\u001b[38;5;0m\u001b[m\u001b(B"]
+[0.029339, "o", "/kuksa-client\u0007\u001b[38;5;0m\u001b[m\u001b(B"]
 [0.029441, "o", "\u001b[38;5;237m⏎\u001b[m\u001b(B                                                                           \r⏎ \r\u001b[K"]
 [0.031639, "o", "Welcome to fish, the friendly interactive shell\r\n"]
 [0.043529, "o", "\u001b[2 q"]
 [0.043648, "o", "\u001b[?2004h"]
-[0.064344, "o", "\u001b]0;fish /home/wenwen/tools/kuksa.val/kuksa_viss_client\u0007\u001b[38;5;0m\u001b[m\u001b(B"]
+[0.064344, "o", "\u001b]0;fish /home/wenwen/tools/kuksa.val/kuksa-client\u0007\u001b[38;5;0m\u001b[m\u001b(B"]
 [0.065271, "o", "> \u001b[K"]
-[0.36832, "o", "python -m kuksa_viss_client"]
-[0.368546, "o", "\u001b[27D\u001b[38;5;26mpython\u001b[38;5;0m\u001b[m\u001b(B \u001b[38;5;39m-m\u001b[38;5;0m\u001b[m\u001b(B \u001b[38;5;39mkuksa_viss_client\u001b[38;5;0m\u001b[m\u001b(B"]
+[0.36832, "o", "python -m kuksa_client"]
+[0.368546, "o", "\u001b[27D\u001b[38;5;26mpython\u001b[38;5;0m\u001b[m\u001b(B \u001b[38;5;39m-m\u001b[38;5;0m\u001b[m\u001b(B \u001b[38;5;39mkuksa_client\u001b[38;5;0m\u001b[m\u001b(B"]
 [1.174898, "o", "\r\n"]
 [1.175056, "o", "\u001b[38;5;0m\u001b[m\u001b(B"]
 [1.175185, "o", "\u001b[2 q"]
 [1.175268, "o", "\u001b[?2004l"]
 [1.175686, "o", "\u001b]0;python /hom"]
 [1.175759, "o", "e/wenwen/tools/kuksa.val/kuksa_viss_clien"]
```

Comment:
viss still appears here and there

(By the way, in the long run we should preferably regenerate/recreate this one, as I think the content is outdated, and we should preferably not include "real usernames" (like wenwen)), but that is for a separate ticket - could possibly be checked by @lukasmittag

Reply:
With export PS1='<name to display> ' you can set what is displayed as name in terminal. I would suggest something like kuksa~. To highlight the name in a specific color you have to add export PS1='\[\033[01;31m\]\kuksa~\[\033[00m\]' for red for example

Reply:
I will have a further look in how the gifs are generated (and will add some for the grpc testclient).